server/controllers/v1/tests/user.test.ts

```diff
@@ -205,7 +205,9 @@ describe('v1 - User Controller', () => {
         fail('nullperson');
       }
 
-      expect(testPerson.confirmed).toBeFalsy();
+      testPerson.confirmed = true;
+      await testPerson.save();
+      expect(testPerson.confirmed).toBeTruthy();
```

Comment:
This isn't really adding any value. Your setting a field to true in the test and ten checking that its true.

Reply:
The truthy thing was sort of a vestigial expect statement, I should have removed it tbh